client/src/component/Search.js

```diff
@@ -0,0 +1,45 @@
+import "./User.css";
+import { useContext } from "react";
+import { AppContext } from "../App";
+
+export function Search() {
+    const { userName, setCodewarsData, setGitHubData } = useContext(AppContext);// we suposed to use context with this name//
+    //as search button clicked this function will happen
```

Comment:
I think these comments can probably be removed. Changing 'searchHandle' to 'handleSearch' may help to explain its use

Reply:
done